sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuDeviceManager.scala

```diff
@@ -315,9 +315,21 @@ object GpuDeviceManager extends Logging {
       }
 
       Cuda.setDevice(gpuId)
-      Rmm.initialize(init, logConf, poolAllocation)
-      RapidsBufferCatalog.init(conf)
+      try {
+        Rmm.initialize(init, logConf, poolAllocation)
+      } catch {
+        case ex: Throwable =>
```

Comment:
Propose changing this to `ai.rapids.cudf.CudfException`, since that's the exception that looks to be thrown.

Reply:
What if a `RmmException` being thrown?